hazelcast/src/test/resources/com/hazelcast/config/yaml/testcases/member/advanced-network/success.json

```diff
@@ -130,7 +130,12 @@
             "port": 5701
           },
           "public-address": "dummy",
-          "reuse-address": true
+          "reuse-address": true,
```

Comment:
can you also add failure tests for these?

Reply:
https://github.com/hazelcast/hazelcast/pull/23319/commits/cbda1671ff96c119bb190acfd420032ecf395e44